src/components/sidepanel/ControlBar.js

```diff
@@ -25,10 +25,13 @@ const PollToggle = styled(ToggleButton)`
   flex: 0;
 `;
 
-@inject(app("Filters", "Update", "Time"))
+@inject(app("Filters", "Update", "Time", "UI"))
 @observer
 class ControlBar extends Component {
-  onClickReset = () => this.props.Filters.reset();
+  onClickReset = () => {
+    this.props.Filters.reset();
+    this.props.UI.hideJourneyGraph();
```

Comment:
No need to explicitly hide the graph on reset. It will be hidden when it doens't have any data and the data disappears when the app is reset. Also, there's a separate function for hooking into the reset event, see the FilterStore.